packages/app/src/actions/court.js

```diff
@@ -2,6 +2,8 @@ import Network from '../web3/Network'
 import ErrorActions from './errors'
 import * as ActionTypes from '../actions/types'
 
+const HEARTBEAT_MAX_TRANSITIONS = 10
```

Comment:
cc @bingen I remember we had a conversation about increasing this to `20`, since it would very safely fit under the gas limit.

Reply:
Yes, I think it was in the context of [services](https://github.com/aragonone/court-backend/blob/master/packages/services/src/workers/heartbeat.js#L5)